### ISSUE TITLE
IP shown as ::ffff:x.x.x.x

### ISSUE THREAD

**Issue by NibeP** [DinoTools/dionaea#143]
##### ISSUE TYPE
 - Bug Report

##### DIONAEA VERSION
```
Dionaea Version 0.6.0
Compiled on Linux/ARM at Dec 14 2017 22:22:53 with gcc 5.4.0 20160609
Started on raspberrypi running Linux/armv7l release 4.9.59-v7+
```

##### CONFIGURATION
listen.mode=getifaddrs

##### OS / ENVIRONMENT
 - Ubuntu 16.04 (Docker container)

##### SUMMARY
Remote IP's are shown like `::ffff:192.168.119.31` instead of `192.168.119.31`

##### STEPS TO REPRODUCE
nmap -p [a blackhole port] [IP of dionaea]
tail -f /opt/dionaea/var/dionaea/dionaea_incident.json

##### EXPECTED RESULTS
Show IP in format like: `x.x.x.x` instead of `::ffff:x.x.x.x`.

##### ACTUAL RESULTS
Remote IP's are shown like `::ffff:192.168.119.31` instead of `192.168.119.31`

**Comment by Petertjuh360** [comment]
Additional information:
This happens in a Docker container. I also have running Dionaea on a CentOS VM where this doesn't happen.

**Comment by NibeP** [comment]
I also detected that a single detection (for example going to http sensor one time) will be reported as 2x tcp.accept.

**Comment by phibos** [comment]
I think this is related to #193

**Comment by phibos** [comment]
This issues should be fixed with PR #208 and will be included in version 0.8.0.
